common/owner_certificate/owner_certificate_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ownercertificate
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"testing"
+
+	_ "embed"
+)
+
+var (
```

Comment:
I think this will break copybara when we import :< I wonder if there's a way to make it automatically pick up the right test files... 🤔

Reply:
Which part specifically, the embedded files? We did a similar thing in https://github.com/openconfig/bootz/blob/6575eb87c559f913274c8e2182085a374b1f702f/common/ownership_voucher/ownership_voucher_test.go#L24.

Reply:
no the list of testdata files. Buildozer isn't currently set up to automatically add them

Reply:
https://github.com/openconfig/bootz/pull/92 should help with this by removing all hardcoded data